src/Components/CreateDog/CreateDog.js

```diff
@@ -0,0 +1,85 @@
+import { useEffect, useState } from "react";
+import "./CreateDog.styles.scss";
+import axios from "axios";
+
+function DogCardInfo() {
+  const [dogName, setDogName] = useState([]);
+  const [dogInfo, setDogInfo] = useState([]);
+
+  useEffect(() => {
+    RandomDogInfo();
+    RandomName();
+    modifiedObject();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [setDogName, setDogInfo]);
+
+  //This function is responsible for getting image, breed Name, size into the cards
+  //However it also saves the rest of the unused data.
+  const RandomDogInfo = async () => {
+    const key = "9/8P1dgeiaMQRHYSg9SgkA==olxKKsYDWHZlEAYU";
```

Comment:
hide your keys

Reply:
resolved the issue with this and have prevented this from happening again